etc/scripts/json2csv.py

```diff
@@ -246,7 +244,7 @@ def cli(json_input, csv_output):
 
     JSON_INPUT is either a ScanCode json format scan or the data.json file from a ScanCode html-app format scan.
 
-    Paths will be prefixed with '/code/' to provide a common base directory for scanned resources.
+    #Paths will be prefixed with '/code/' to provide a common base directory for scanned resources.
```

Comment:
@jack17529 No. This option, which for now I will call `--prepend-root`, should prepend the path value with `/code` when the option is called. When the `--prepend-root` option is not called, the path value should not be modified.